README.md

```diff
@@ -227,6 +227,10 @@ $ puma -b 'ssl://localhost:9292' config.ru
 $ puma -b ssl://localhost:9292 -b tcp://localhost:9393 config.ru
 ```
 
+There is a known limitation to the [`localhost`] gem in Puma. Puma can't start in cluster 
+mode when the user didn't preload the app or requires the [`localhost`] gem in a 
+configuration file.
```

Comment:
I don't think we need this paragraph, instead we should change the documentation above to say that you need to require `localhost` in your Puma config and not `config.ru` – that should cover all scenarios, right?

Reply:
See https://github.com/puma/puma/tree/87c052f514488286a9ee70855db8a265c90a4dbb#self-signed-ssl-certificates-via-the-localhost-gem-for-development-use

We should also change this line

>Puma automatically configures SSL when the [localhost](https://github.com/socketry/localhost) gem is loaded in a development environment:

to

> Puma automatically configures SSL when the [localhost](https://github.com/socketry/localhost) gem is loaded

because as far as I can see, there's nothing limiting use of `localhost` to just development (and that is fine IMHO)

Reply:
I changed the docu as suggested